src/renderers/webvr/WebXRManager.js

```diff
@@ -77,7 +77,19 @@ function WebXRManager( renderer ) {
 	this.setDevice = function ( value ) {
 
 		if ( value !== undefined ) device = value;
-		if ( value instanceof XRDevice ) gl.setCompatibleXRDevice( value );
+		if ( value instanceof XRDevice ) {
+
+			if ( gl.setCompatibleXRDevice !== undefined ) {
+
+				gl.setCompatibleXRDevice( value );
+
+			} else {
+
```

Comment:
Do you mind adding this comment here (and in all the other `else`s)?

```
// TODO: Remove when Chrome 73 reaches stable
```

Reply:
@mrdoob Yep, I can accomodate / test latest Canary changes. Do you want to merge? @jsantell recommended to maybe wait until the dust settles. It's true there's a fair amount of churn at the moment. I let you decide.

Reply:
https://github.com/mrdoob/three.js/pull/15604#discussion_r257491150